src/omv/sensors/paj6100.h

```diff
@@ -9,6 +9,7 @@
 #ifndef __PAJ6100_H__
 #define __PAJ6100_H__
 #if (OMV_ENABLE_PAJ6100 == 1)
+#include "sensor.h"
```

Comment:
https://github.com/openmv/openmv/pull/1365#discussion_r652977159
You remove it, I add it back.